### ISSUE TITLE
[Feature Request] Strongly Required -> Combobox for "Default, Track, No Track, ..." for each Category and Expansion

### ISSUE THREAD

**Issue by fubaWoW** [Urtgard/WQAchievements#148]
Strongly Requires a Combobox for "Default, Always Track, Don't Track, ..." for each Category and Expansion!

It's more than just annoying to Disable, Enable or even Switch EVERY SINGLE Achievement, Pets, Quest, Toy, etc...

would be very nice you can include this!

**Comment by Urtgard** [comment]
I agree, that would be a good addition. But I don't think I'll have time to add that any time soon.
